pages/login.tsx

```diff
@@ -5,60 +5,114 @@ import {
     Text,
     Input,
     FormControl,
+    useControllableState,
 } from "@chakra-ui/react";
+import { useState } from "react";
+import useLocalStorage from "@utils/useLocalStorage";
+import { signIn } from "next-auth/client";
+import Wrapper from "@components/SDCWrapper";
+import isEmail from "validator/lib/isEmail";
 
 /**
  * This is the page that a user will use to either login or register
  * to the SDC platform
  */
 export default function Login(): JSX.Element {
+    // hook for storing the email
+    const [email, setEmail] = useState("");
+    // save the email into localstorage for email verification page
+    const [, setLocalStorageEmail] = useLocalStorage(
+        "sdc-email-verification",
+        "",
+    );
+    const [value, setValue] = useControllableState({ defaultValue: false });
     return (
-        <Center h="500px">
-            <Box width="700px">
-                <Center>
-                    <Text fontWeight="700" fontSize="32px">
-                        Sign Up Or Sign In
-                    </Text>
-                </Center>
-                <Center>
-                    <Text fontWeight="400" fontSize="18px" mt="20px">
-                        Registration for Summer 2021 classes begin June 31, 2021
-                    </Text>
-                </Center>
-                <FormControl id="email">
+        <Wrapper>
+            <Center h="500px">
+                <Box width="700px">
+                    <Center>
+                        <Text fontWeight="700" fontSize="32px">
+                            Sign Up Or Sign In
+                        </Text>
+                    </Center>
+                    <Center>
+                        <Text fontWeight="400" fontSize="18px" mt="20px">
+                            Registration for Summer 2021 classes begin June 31,
+                            2021
+                        </Text>
+                    </Center>
+                    <FormControl id="email">
+                        <Center>
+                            <Input
```

Comment:
could you make the font weight a bit lighter?

Reply:
which text do you want me to make lighter?

Reply:
The email text input and also maybe the "Registration for Summer 2021 ..." text above, using the figma design as reference https://www.figma.com/file/ad7RNFErTGP7HkL52edTwQ/SDC-External-Application?node-id=1171%3A15116

![image](https://user-images.githubusercontent.com/42329517/126858303-24cdc58d-c6a7-4837-ad02-5e1276b21536.png)

Reply:
Update - we will fix this in a separate PR